processor/tailsamplingprocessor/processor_decisions_test.go

```diff
@@ -386,8 +385,7 @@ func TestLateArrivingSpanUsesDecisionCache(t *testing.T) {
 	// The final decision SHOULD be Sampled.
 	require.EqualValues(t, 1, nextConsumer.SpanCount())
 
-	// Drop the trace to force cache to make decision
-	tsp.dropTrace(traceID, time.Now())
```

Comment:
@jpkrohling I was able to reuse the existing decision tests for the new functionality 👍 It was meant to be 😄